certified-connectors/gofileroom/apiProperties.json

```diff
@@ -1,80 +1,93 @@
-        "templateId": "setheader",
-        "title": "Set Authorization Header"
-      }
-    ]
-  }
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFBF10",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "setheader",
+        "title": "Set Authorization Header",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "Authorization",
+          "x-ms-apimTemplateParameter.value": "Basic @headers('X-Authorization')",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request",
+          "x-ms-apimTemplate-operationName": [
+            "CreateGroup",
+            "SetGroupDocSecurity",
+            "ModifyGroup",
+            "GetGroupPermissions",
+            "SetGroupPermissions",
+            "GetGroupDocumentSecurity",
+            "GetGroups",
+            "CreateUsers",
+            "DeleteUser",
+            "SetUserDocSecurity",
+            "GetUserInfo",
+            "GetLicenses",
+            "ModifyUser",
+            "GetPasswordPolicy",
+            "SetUserPermissions",
+            "GetListOfReports",
+            "GetUploadLocation",
+            "GetUserDocumentSecurity",
+            "GetUserPermission",
+            "GetUsers",
+            "GetLookupList",
+            "CreateDocument",
+            "CopyDocument",
+            "GetDocumentStatus",
+            "MergePDF",
+            "DocumentSearch",
+            "DocumentReindex",
+            "TaxsortDocument",
+            "DocumentDelete",
+            "GetDocument",
+            "GetDocumentHistory",
+            "GetDocumentIndexes",
+            "PublishDocumentStatus",
+            "GetDrawers",
+            "GetDrawerIndexes",
+            "GetFirmFlowDeliverableReport",
+            "ValidateIndexes",
+            "GetDynamicRulesForIndex",
+            "IndexLookupListFind",
+            "GetListTypeIndexData",
+            "GetChildIndexes",
+            "Logout",
+            "ValidateToken",
+            "CreateWorkflow",
+            "DeleteMasterDeliverable",
+            "DeleteWorkflow",
+            "DeleteWorkflows",
+            "EditMasterDeliverable",
+            "EditWorkflow",
+            "GetMasterDeliverable",
+            "GetNotes",
+            "RouteWorkflowV2",
+            "TrackingReportByWorkflow",
+            "AddMasterDeliverable",
+            "GetUserInfoV2",
+            "TrackingReportByDeliverableV2"
+          ]
+        }
+      }
+    ],
+    "publisher": "Teribery, Dustin (Tax&Accounting Prof)",

```

Comment:
@AlfredoDinunzioTR The given publisher name looks like an individual name which is not allowed. It should be an organization or company name for "certified-connectors"